vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -326,6 +339,36 @@ public Builder setLogLevel(LogLevel level) {
             return this;
         }
 
+        public Builder addNetworkInterceptor(Interceptor interceptor) {
+            if (interceptor != null) {
+                this.networkInterceptors = new ArrayList<>();
+                this.networkInterceptors.add(interceptor);
```

Comment:
I would say this behavior needs to be modified. As a user, I may want to call ```addNetworkInterceptor``` then ```addNetworkInterceptor``` again. With this logic, the first interceptor I add would be gone.